Sources/VaporJWT/Request+JWT.swift

```diff
@@ -0,0 +1,50 @@
+import HTTP
+import JWT
+
+extension Request {
+    /// Parses and returns a JWT from the request
+    /// if one exists.
+    /// The JWT will also be verified with the supplied
+    /// signer.
+    public func jwt(verifyUsing signer: Signer) throws -> JWT {
```

Comment:
My thinking here was trying to shield the user from ever getting an unverified JWT. For someone that's just learning JWT that seems like a somewhat easy mistake to make. I don't see any use case as a framework developer to have access to an unverified JWT. It just seems like a security risk. 

But I'm also fairly new to JWT so maybe I'm missing something here. 

Possibly a compromise could be having `Signer?` in the signature so you can skip verification but it's a little more obvious, i.e:

```swift
let jwt = try request.jwt(verifyUsing: nil)
```